README.md

```diff
@@ -1,12 +1,44 @@
-# OpenTelemetry Instrumentation for Java
-
-## Join the discussions!
-
-* Watch this repo :eye:
-* Join the [Gitter channel](https://gitter.im/open-telemetry/opentelemetry-java-instrumentation)
-* Join the [weekly meeting](https://github.com/open-telemetry/community#java-instrumentation)
-
-## Introduction
+---
+
+<p align="center">
+  <strong>
+    <a href="https://github.com/open-telemetry/opentelemetry-java-instrumentation#getting-started">Getting Started<a/>
+    &nbsp;&nbsp;&bull;&nbsp;&nbsp;
+    <a href="https://github.com/open-telemetry/community#special-interest-groups">Getting Involved<a/>
+    &nbsp;&nbsp;&bull;&nbsp;&nbsp;
+    <a href="https://gitter.im/open-telemetry/opentelemetry-java-instrumentation">Getting In Touch<a/>
+  </strong>
+</p>
+
+<p align="center">
+  <a href="https://goreportcard.com/report/github.com/open-telemetry/opentelemetry-java-instrumentation">
+    <img alt="Go Report Card" src="https://goreportcard.com/badge/github.com/open-telemetry/opentelemetry-java-instrumentation?style=for-the-badge">
+  </a>
```

Comment:
> There was an error processing your request: Could not analyze the repository: no .go files found

I don't think that we need this one - there are no go files in this repo 😄

Reply:
Yup fixing